src/blog/2024/10/exploring-flowfuse-sbom-feature.md

```diff
@@ -0,0 +1,68 @@
+---
+title: FlowFuse Software's bills of material helps enhance Application Security and Management
+subtitle: Enhancing the Security of Your Node-RED Applications
+description: Learn how the FlowFuse SBOM feature improves the security and management of Node-RED applications by tracking dependencies and identifying vulnerabilities.
+date: 2024-10-14
+authors: ["sumit-shinde"]
+image: 
+tags:
+ - post
+ - flowfuse
+ - enhancing node-red security
+ - flowfuse software bills of material
+---
+
+FlowFuse recently launched its Software Bill of Materials (SBOM) feature for enterprise customers. This powerful tool enhances security and management within projects, particularly in the Node-RED ecosystem. As open-source libraries and software continue to play a pivotal role in the industry, monitoring third-party components used in projects becomes essential. The SBOM enables organizations to track dependencies and identify vulnerabilities, ensuring compliance and mitigating risks.
+
+<!--more-->
+
+In this article, we will explore the SBOM feature and discuss how it impacts security and management in Node-RED applications.
+
+## What is an SBOM, and How Does It Enhance Security?
+
+A Software Bill of Materials (SBOM) is a detailed list of all the components that make up a software application, just like how a bill of materials for a physical product lists every part used in its construction, an SBOM provides a breakdown of all the software libraries, packages, and dependencies used in a project.
+
+This transparency is crucial for security, allowing developers and organizations to track what’s inside their software. By knowing precisely what components are in use, you can quickly identify outdated or vulnerable dependencies that may pose security risks. An SBOM helps you monitor third-party nodes, ensuring that any related security issues can be addressed promptly and reducing the chance of vulnerabilities being exploited.
+
+## Exploring the FlowFuse SBOM Feature
+
+**[FlowFuse](/)** is a platform that makes it easy to remotely manage multiple Node-RED instances from one place. It helps teams collaborate efficiently, share resources, and scale applications with a user-friendly interface. FlowFuse also includes strong security features to prevent unauthorized access and advanced controls to manage the security of each team and application, allowing you to group and manage your Node-RED applications effectively.
```

Comment:
Let's move away from "Remote" in favour of "Edge". I also feel like this "pitch" might not have a place in this article, and even if you include it it should be more in line with the FF messaging page in the handbook: https://flowfuse.com/handbook/customer/marketing/messaging/#messaging